Ruby/fibonacci/fibonacci.rb

```diff
@@ -1,5 +1,24 @@
-def fibonacci( n )
+def FibonacciSequence( n )
   return  n  if ( 0..1 ).include? n
-  ( fibonacci( n - 1 ) + fibonacci( n - 2 ) )
+  ( FibonacciSequence( n - 1 ) + FibonacciSequence( n - 2 ) )
 end
-puts fibonacci( 5 )
+
+DEFAULT = 10
+
+n = DEFAULT
+
+if ARGV.length > 0
+  n = ARGV[0].to_i
+else
+  puts "Which Fibonacci index to find? "
+  n = ARGF.gets.to_i
+end
+
+if n < -1
+  n = DEFAULT
+elsif n > 40
```

Comment:
If we cut off, then we should cut off at the same index everywhere consistently and log it